acra/configuring-maintaining/logging/formats.md

```diff
@@ -0,0 +1,90 @@
+---
+weight: 1
+title: Logging output
+bookCollapseSection: true
+---
+
+# Logging output
```

Comment:
I still think the title should be “logging formats”, or something like this. Consistent with the URL of this page, and I don't remember seeing a section called “logging output” where ‘logging’ is used as noun, not a verb.